ui/src/actions/sampleview.js

```diff
@@ -336,92 +310,47 @@ export function sendGoToBeam(x, y) {
   };
 }
 
-export function sendStartAutoCentring() {
-  return () => {
-    fetch('/mxcube/api/v0.1/sampleview/centring/startauto', {
-      method: 'PUT',
-      credentials: 'include',
-      headers: {
-        Accept: 'application/json',
-        'Content-type': 'application/json',
-      },
-    }).then((response) => {
-      if (response.status >= 400) {
-        throw new Error('Server refused to start autocentring');
-      }
+export function addShape(shapeData = {}, successCb = null) {
+  return async (dispatch) => {
+    try {
+      const json = await sendAddOrUpdateShapes([shapeData]);
```

Comment:
Note the use of the same fetch function `sendAddOrUpdateShapes` in both `addShape` and `updateShapes`.